### ISSUE TITLE
Mention template functionality in documentation

### ISSUE THREAD

**Issue by lukket** [medialesson/hugo-theme-event#38]
**What functionality is missing? What could be better?**
In the documentation, it's written that one could fork the demo repository to get started.

**Describe the solution you'd like (acceptance criteria)**
Better than forking is to use GitHub's template functionality because it doesn't keep the whole commit history. Therefore, the demo project is already a template repository. It's just not documented yet.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by NataliiaPodp** [comment]
@lukket is #67  not a duplication of the current issue?

**Comment by lukket** [comment]
No, this here is about getting started if you want to use the theme (using the template functionality of the demo project). #67 is about how to contribute changes to the actual theme, which is easiest done by cloning the demo project and then committing to the theme via Git's submodule.
